### DISCUSSION TITLE
Select value with dots (`.`) in inline `theme` function

### DISCUSSION THREAD

**Discussion by ajmnz** [ben-rogerson/twin.macro#814] [Q&A]
I'm trying to access the `2.5` property of `spacing` in my theme inside the inline `theme` function, but it doesn't seem to get picked up.

```tsx
<Flex tw="flex-col pt-1 pl-[calc(20px + theme(spacing.2.5))]"></Flex>
```

Tried escaping the dot `theme(spacing.2\.5)`, but no luck. Is there any way to do such thing?

Thanks!

**Comment by ben-rogerson** [comment]
Sure can - you'll just need to use brackets for values with a dot in them - `theme(spacing[2.5])`

**Reply by ben-rogerson** [reply]
I've created [a fix](https://github.com/ben-rogerson/twin.macro/pull/815) which will be in the next twin version.

**Reply by ajmnz** [reply]
Square brackets work good enough, but thanks for the fix either way!
